lib/easypost/services/api_key.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+# An ApiKey allows you to authenticate with the EasyPost API.
+class EasyPost::Services::ApiKey < EasyPost::Services::Service
+  # Retrieve all api keys.
+  def all
+    @client.make_request(:get, 'api_keys', EasyPost::Models::ApiKey)
```

Comment:
This endpoint doesn't actually return a list of API key objects. Doesn't matter since we're not strictly deserializing.